.github/CONTRIBUTING.md

```diff
@@ -117,7 +131,7 @@ Time: 16.46 seconds, Memory: 48.00MB
 OK (12 tests, 0 assertions)
 ```
 
-If you didn't install PHPCS/WPCS/PHPUnit using Composer, you will need to type the above command in to run the unit tests. Make sure you replace the path to PHPUnit and the path to PHPCS when you do.
+If you didn't install PHPCS/WPCS/PHPUnit using Composer, you will need to type the above command in to run the unit tests. Make sure you replace the path to PHPUnit and the path to PHPCS when you do, and add the `--bootstrap="/path/to/PHPCS/tests/bootstrap.php"` part when running the unit test.
```

Comment:
... and, _if you didn't setup your own `phpunit.xml` file_, add `--bootstrap="/path/to/PHPCS/tests/bootstrap.php"`.